### ISSUE TITLE
Allow substitution based on poster's privileges

### ISSUE THREAD

**Issue by tms13** [Benjol/SE-AutoReviewComments#125]
I have a stock comment for weak answers that should really be comments:
```
###[A] No code in answer
While this might be a valuable hint to solve the problem, an answer really needs to demonstrate the solution.  Please [edit] to provide example code to show what you mean.
Alternatively, consider writing this as a comment instead.
```
As this so often applies to new users, they may well not have the privileges to comment, so this message isn't exactly appropriate.  I'd love to be able to have alternatives based on their privileges:
```
[can-comment?]{Alternatively, consider writing this as a comment instead.}
```
and/or
```
[!can-comment?]{When you have earned enough reputation, you will be able to comment on this post.}
```

**Comment by Benjol** [comment]
This seems a bit feature-creepy to me, you're welcome to fork and implement your own version.
Sorry if that seems like a cop-out.

**Comment by tms13** [comment]
Fair point.  I do like the similar feature in the review queues, where users who can't yet comment get a message to stick to questions that don't need clarification until they have sufficient privileges.  I'm unlikely to find time to work on this, and it doesn't bother me too much, so I guess the idea will just languish until somebody else picks it up.

**Comment by 501st-alpha1** [comment]
Would it be less feature-creepy (or otherwise better) if the condition applied to the entire comment? E.g.:

    ###[A] No code in answer (>50)
    While this might be a valuable hint to solve the problem, an answer really needs to demonstrate the solution.  Please [edit] to provide example code to show what you mean.
    Alternatively, consider writing this as a comment instead.

    ###[A] No code in answer (<50)
    While this might be a valuable hint to solve the problem, an answer really needs to demonstrate the solution.  Please [edit] to provide example code to show what you mean.

**Comment by Benjol** [comment]
@501st-alpha1 that looks a bit better. I'll mull it over but must admit I'm a bit 'booked' at the moment so time not much free to work on it...

**Comment by Benjol** [comment]
@501st-alpha1 that looks a bit better. I'll mull it over but must admit I'm a bit 'booked' at the moment so time not much free to work on it...
